docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
-    * This means it gets an additional bonus between 2.67% and 4%
-    * This means the block proposer’s bonus is between 3.67% and 5% of the total block rewards.
+#### Downtime event
 
-    4. 20% goes to the community pool
-    5. The rest is distributed to all of the bonded validators (including the block proposer) proportional to their voting power, regardless of them signing the block.
+If a validator fails to sign 90%  of blocks within a 19200-block interval (i.e. 17280 blocks), then it is jailed and the following occurs:
 
+    1. It becomes not-bonded and begins unbonding
+    2. It stops signing and proposing blocks
+    3. It stops earning block rewards
+    4. Its stake gets slashed by 0.01%
 
-(In addition, it gets (4*P)% percent of the block rewards where P=(total power of validators with included precommits / total bonded validator power)
-This means it gets an additional bonus between 2.67% and 4%
-This means the block proposer’s bonus is between 3.67% and 5% of the total block rewards.)
+The validator must unjail itself as quickly as possible after being jailed. 
 
-## Distribution Schedule
+#### Double signing event
+ 
+If a validator signs more than one block at the same height then 5% of their tokens are slashed. 
+This could be the case if someone duplicates their validator and tries to double their voting power.
 
-**Tx fees** are ideally the main source of revenue for validators and delegators. However, due to the expectation of low early engagement numbers on the Cudos blockchain, we will also provide staking rewards to users.
+#### What happens to slashed tokens?
 
-* We will distribute a total of ~1,5 billion tokens as staking rewards in ~10 years.
+Slashed tokens are sent to the community pool.
 
-* The exact number of tokens is given by the formula 
+:::warning Slashing
+When a validator’s tokens are slashed by a certain percentage, so are its delegators.
+:::
 
-**f(x) = 1.8 x^2 - 53 x + 358** 
+#### How to unjail
 
-**where *x* is in years and *f(x)* is in millions of CUDOS.**
+To unjail, the validator must send a transaction from the jailed validator account via the `Cudos-noded CLI`. 
 
-* Hence the amount to be distributed in a given time interval is obtained by integrating the above function over that interval.
+```shell
+cudos-noded tx slashing unjail --from mykey [flags]
+```
 
-* Due to the assumption that tx fees will eventually be the main source of revenue, staking rewards start high and then decrease quadratically in time.
+### Unbonding period
 
-:::tip Important
+If a validator is jailed or leaves the bonded validators set, it enters an **unbonding period**. A validator that has failed to get itself back into the bonded validators set *within 21 days* is said to be **unbonded**.
 
-Important note: Validators need to use the --commission flag in the CLI in order to withdraw their commission.
+:::caution 
+Slashing can still occur during the unbonding period.
```

Comment:
Might be a good idea to clarify a bit by saying something like "Validators are still subject to slashing upon evidence of bad behaviour while they were previously bonded". Otherwise, users might be worried if they would get slashed for not signing blocks during the unbonding period, which won't happen.

Reply:
Done